include/mesh.tcc

```diff
@@ -69,27 +69,26 @@ bool mpm::Mesh<Tdim>::remove_node(
 template <unsigned Tdim>
 template <typename Toper>
 void mpm::Mesh<Tdim>::iterate_over_nodes(Toper oper) {
-  tbb::parallel_for(
-      tbb::blocked_range<int>(size_t(0), size_t(nodes_.size()),
-                              tbb_grain_size_),
-      [&](const tbb::blocked_range<int>& range) {
-        for (int i = range.begin(); i != range.end(); ++i) oper(nodes_[i]);
-      },
-      tbb::simple_partitioner());
+  tbb::parallel_for(tbb::blocked_range<int>(size_t(0), size_t(nodes_.size()),
```

Comment:
Please run clang-format using `clang-tools/format.sh`

Reply:
@kks32 I think we need to check this. The `clang-tools/format.sh` always return the correction like @ezrayst has. You saw it in my previous PR too...

Reply:
You can see it fail in `circleci` https://circleci.com/gh/cb-geo/mpm/3979?pipelines-ui-opt-out, so it's a local issue

Which version of clang-tools do you have?

Reply:
```
ezrayst@ezrayst-acer:~/Documents/cb-geo/mpm/build$ clang-format --version
clang-format version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
```

Reply:
I am having the same version as ezra

Reply:
Could you please upgrade to the later versions: `clang-format version 9.0.1`